From 374c91e07c0996d5aacb25ba71b3aa1fcf83afec Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Amadeusz=20=C5=BBo=C5=82nowski?= <aidecoe@aidecoe.name>
Date: Sun, 4 Jul 2010 21:21:29 +0200
Subject: [PATCH 2/2] dm, dmraid, lvm: install udev rules file 64-device-mapper.rules

Gentoo ebuild for LVM2 prior to 2.02.63-r1 doesn't install rules:
  10-dm.rules
  11-dm-lvm.rules
  13-dm-disk.rules
  95-dm-notify.rules
but provides only 64-device-mapper.rules combining more or less the
above rules files.
---
 modules.d/90dm/install     |    3 +++
 modules.d/90dmraid/install |    3 +++
 modules.d/90lvm/install    |    3 +++
 3 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/modules.d/90dm/install b/modules.d/90dm/install
index f77d945..0bc9ced 100755
--- a/modules.d/90dm/install
+++ b/modules.d/90dm/install
@@ -7,3 +7,6 @@ inst dmsetup
 
 inst dmeventd
 inst_rules 10-dm.rules 95-dm-notify.rules
+# Gentoo ebuild for LVM2 prior to 2.02.63-r1 doesn't install above rules
+# files, but provides the one below:
+inst_rules 64-device-mapper.rules
diff --git a/modules.d/90dmraid/install b/modules.d/90dmraid/install
index a016e4d..81c63f0 100755
--- a/modules.d/90dmraid/install
+++ b/modules.d/90dmraid/install
@@ -15,6 +15,9 @@ done
 
 
 inst_rules 10-dm.rules 13-dm-disk.rules 95-dm-notify.rules 
+# Gentoo ebuild for LVM2 prior to 2.02.63-r1 doesn't install above rules
+# files, but provides the one below:
+inst_rules 64-device-mapper.rules
 
 inst "$moddir/dmraid.sh" /sbin/dmraid_scan
 
diff --git a/modules.d/90lvm/install b/modules.d/90lvm/install
index 3e20866..461d1a7 100755
--- a/modules.d/90lvm/install
+++ b/modules.d/90lvm/install
@@ -13,6 +13,9 @@ if [[ $hostonly ]] || [[ $lvmconf = "yes" ]]; then
 fi
 
 inst_rules 10-dm.rules 13-dm-disk.rules 95-dm-notify.rules 11-dm-lvm.rules
+# Gentoo ebuild for LVM2 prior to 2.02.63-r1 doesn't install above rules
+# files, but provides the one below:
+inst_rules 64-device-mapper.rules
 
 inst "$moddir/lvm_scan.sh" /sbin/lvm_scan
 inst_hook cmdline 30 "$moddir/parse-lvm.sh"
-- 
1.7.2

